content/en/docs/setup/getting-started/index.md

```diff
@@ -9,46 +9,30 @@ aliases:
 keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
-This guide is intended for users who are new to Istio and lets you quickly evaluate Istio by installing the
-`demo` [configuration profile](/docs/setup/additional-setup/config-profiles/).
+This guide lets you quickly evaluate Istio.
 
-If you are already familiar with Istio or interested in installing other configuration profiles
-or a more advanced [deployment model](/docs/ops/deployment/deployment-models/),
-follow the [installing with {{< istioctl >}} instructions](/docs/setup/install/istioctl) instead.
+These steps assume you are using
+[Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) and have a
+{{< gloss >}}cluster{{< /gloss >}} running a compatible version of Kubernetes.
+Minikube is a tool that makes it easy to run Kubernetes locally with a
+single-node Kubernetes cluster inside a Virtual Machine (VM) on your computer.
 
-{{< warning >}}
-The demo configuration profile is not suitable for performance evaluation. It
-is designed to showcase Istio functionality with high levels of tracing and
-access logging.
-{{< /warning >}}
-
-1. [Set up your platform](#platform)
-1. [Download the release](#download)
-1. [Install Istio](#install)
-1. [Enable automatic sidecar injection](#enable-injection)
-1. [Deploy the Bookinfo sample application](#Bookinfo)
-
-## Set up your platform {#platform}
+To use a cluster on a different platform or in the cloud, follow the appropriate
+[platform-specific setup instructions](/docs/setup/platform-setup/) before you
+continue.
 
-Before you can install Istio, you need a {{< gloss >}}cluster{{< /gloss >}} running a compatible version of Kubernetes.
-Istio {{< istio_version >}} has been tested with Kubernetes releases {{< supported_kubernetes_versions >}}.
+Follow these steps to get started with Istio:
 
-Create a cluster by selecting the appropriate [platform-specific setup instructions](/docs/setup/platform-setup/).
+1. [Install Istio](#download)
+1. [Deploy the sample application](#bookinfo)
+1. [Open the application to outside traffic](#ip)
+1. [View the dashboard](#dashboard)
 
-Some platforms provide a {{< gloss >}}managed control plane{{< /gloss >}} which you can use instead of
-installing Istio manually. If this is the case with your selected platform, and you choose to use it,
-you will be finished installing Istio after creating the cluster, so you can skip the following instructions.
-For more information, see your platform service provider's documentation.
-
-## Download and install Istio {#download}
-
-Download the Istio release which includes installation files, samples, and the
-[{{< istioctl >}}](/docs/reference/commands/istioctl/) command line utility.
+## Install Istio {#download}
```

Comment:
This section (#download) is referenced from all the other more advanced install instructions, so you can't change it to do install as well.

Reply:
```suggestion
## Download Istio {#download}
```

Reply:
fixed